test/diameter_avp_filter_SUITE.erl

```diff
@@ -82,6 +82,8 @@
 		    #{handler => 'ergw_aaa_ro'}},
 	  apps =>
 	      #{default =>
+		    #{'Origin-Host' => <<"dummy.host">>,
+		      procedures =>
 		    #{init => [#{service => <<"Default">>}],
 		      authenticate => [],
 		      authorize => [],
```

Comment:
the content of `procedures` needs to be reindent.

Reply:
ok